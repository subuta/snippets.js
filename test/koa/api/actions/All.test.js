/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import AllAction from 'lib/koa/api/actions/All'

describe('koa/api/actions/All', () => {
  it('should create koa action', () => {
    expect(format(AllAction({
      model: 'user',
      routeConfig: {}
    }))).toMatchSnapshot()
  })
})
