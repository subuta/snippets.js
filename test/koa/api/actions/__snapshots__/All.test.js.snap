// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`koa/api/actions/All should create koa action 1`] = `
"user.get('/', async (ctx) => {
  const {User} = ctx.state.models
  let params = {}

  /* mat Before index [start] */
  /* mat Before index [end] */

  ctx.body = await User.query()
    .eager('')
    .where(params)
})
"
`;
